quick_purchase/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Quick Purchase
+==============
+This module gives the possibility
+to add easily product into the purchase order (mass line add/update).
+
```

Comment:
Ok. I'll add some screen shot for more precision. 
What did you mean about "Not sure it's sufficient your module" ? There is some use cases that can I add to this module ?